pycbc/workflow/jobsetup.py

```diff
@@ -1723,3 +1724,31 @@ class PycbcSplitBankXmlExecutable(PycbcSplitBankExecutable):
 
     extension='.xml.gz'
 
+class PycbcConditionStrainExecutable(Executable):
+    """ The class responsible for creating jobs for pycbc_condition_strain. """
+
+    current_retention_level = Executable.ALL_TRIGGERS
+    def __init__(self, cp, exe_name, ifo=None, out_dir=None, universe=None,
+            tags=None):
+        super(PycbcConditionStrainExecutable, self).__init__(cp, exe_name, universe,
+              ifo, out_dir, tags)
+
+    def create_node(self, input_files, tags=None):
+        if tags is None:
+            tags = []
+        node = Node(self)
+        start_time = self.cp.get("workflow", "start-time")
+        end_time = self.cp.get("workflow", "end-time")
+        node.add_opt('--gps-start-time', start_time)
+        node.add_opt('--gps-end-time', end_time)
+        node.add_input_list_opt('--frame-files', input_files)
+
+        out_file = File(self.ifo, "gated",
+                        segments.segment(int(start_time), int(end_time)),
+                        directory=self.out_dir, store_file=self.retain_files,
+                        extension=input_files[0].name.split('.', 1)[-1],
+                        tags=tags)
+        node.add_output_opt('--output-strain-file', out_file)
+
```

Comment:
Hi @a-r-williamson, your comment made me think that it is wise to add
```
out_gates_file = File([…])
node.add_output_opt('--output-gates-file', out_gates_file)
```
here so that we can save the gating information to a textfile (out_gates_file).  This should then be stuck in the pyGRB output page to make it easier to spot glitch-related triggers we may gate even with autogating.